static/app/components/updatedEmptyState.tsx

```diff
@@ -0,0 +1,368 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import waitingForEventImg from 'sentry-images/spot/waiting-for-event.svg';
+
+import ButtonBar from 'sentry/components/buttonBar';
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import {GuidedSteps} from 'sentry/components/guidedSteps/guidedSteps';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {PlatformKey, Project, ProjectKey} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import FirstEventIndicator from 'sentry/views/onboarding/components/firstEventIndicator';
+
+type GuidedStepInfo = {
+  install: string;
+  installCode: string;
+  configure?: string;
+  configureCode?: (dsn: string) => string;
+  sourcemaps?: string;
+  sourcemapsCode?: string;
+  verify?: string;
+  verifyCode?: string;
+};
+
+const GuidedStepsMap: Partial<Record<PlatformKey, GuidedStepInfo>> = {
```

Comment:
Is there any way we can somehow re-use the onboarding values defined in `gettingSTartedDocs/*`? I can see these becoming out of sync at some point

Reply:
i think after the trial with the 4 projects, we need to figure out how to generalize this. i think we will eventually move towards using the onboarding values somehow, but for now, we are showing slightly different instructions (mainly streamlining the instructions) so i don't think it's worth the work to align them right now, but agree that this will need to be addressed in the future.

Reply:
Yes that's fair! Just so long as we are aware and keeping tabs on it